html/index.html

```diff
@@ -75,9 +74,75 @@ <h2>Software developer</h2>
       </div>
     </section>
 
+    <section class="works">
+      <div class="works_head">
+        <h2 class="work_title">MY RECENT WORK</h2>
+        <img alt="divider vector" src="../img/Vector.svg" class="divider_c">
+      </div>
+      <div class="works_container">
+        <div class="project">
+          <div class="work_img"></div>
+          <h3 class="works_titles">Multi-Post Stories</h3>
+          <p class="project_st">A daily selection of privately personalized reads;no accounts or sign-ups required. has been the industry's 
+             standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+          <div class="work_tools_tags">
+            <div class="tool_tag">css</div>
+            <div class="tool_tag">html</div>
+            <div class="tool_tag">bootstrap</div>
+            <div class="tool_tag">Ruby</div>
+          </div>
```

Comment:
- [ ] Please note that the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") should be a list `ul` and each of them a `li` 😉